bench/workgen/workgen.cxx

```diff
@@ -189,39 +187,43 @@ int
 WorkloadRunner::start_table_idle_cycle(WT_CONNECTION *conn)
 {
     WT_SESSION *session;
-    WT_CURSOR *cursor;
-    uint64_t start, stop, last_interval;
-    int ret, cycle_count;
-    char uri[BUF_SIZE];
-
-    cycle_count = 0;
-    if ((ret = conn->open_session(conn, NULL, NULL, &session)) != 0) {
-        THROW("Error Opening a Session.");
+    if (conn->open_session(conn, nullptr, nullptr, &session) != 0) {
+        THROW("Error opening a session.");
     }
 
-    for (cycle_count = 0; !stopping; ++cycle_count) {
+    int ret;
```

Comment:
I would suggest moving 'int ret' into the loop, and initialising it, as well as moving 'uri' into the loop too.

Reply:
Good suggestion, fixed in 35adee12db6765de98568b6b01b91f78bddeed2e